scripts/world_bank/wdi/output/WorldBank_StatisticalVariables.mcf

```diff
@@ -634,3 +634,163 @@ statType: dcs:measuredValue
 measuredProperty: dcs:amount
 transferType: dcs:OutwardRemittance
 
+
+Node: dcid:WorldBank/SH_DYN_MORT
```

Comment:
Isn't this the same as the existing SV: ``dcid:MortalityRate_Person_Upto4Years_AsFractionOf_Count_BirthEvent_LiveBirth`` If so, then why the change?